tools/bumper-bot/parser/parser.go

```diff
@@ -0,0 +1,48 @@
+package parser
```

Comment:
We don't need new package for this this way we don't need to make functions public.

Reply:
ok I'll move it to the main package